packages/nimble-components/src/table-column/text-base/group-header-view/index.ts

```diff
@@ -15,7 +19,30 @@ export abstract class TableColumnTextGroupHeaderViewBase<
 
     /**
      * Text to render in the cell.
+     *
+     * The value is initialized to `tableGroupRowNoValuePlaceholder` because if the group
+     * row never has a value defined on it, the change handlers may never get called but
+     * the text needs to be correct.
      */
     @observable
-    public text = '';
+    public text = tableGroupRowNoValuePlaceholderLabel.getValueFor(this);
+
+    /**
+     * Sets `this.text` to the appropriate placeholder if `groupHeaderValue` warrants it.
+     * @param groupHeaderValue The value for the group
+     * @returns `true` if `this.text` was set to a placeholder, `false` otherwise.
+     */
+    protected applyPlaceholderTextIfNeeded(
```

Comment:
I don't like that derived classes have to "opt in" to this behavior by calling it directly. I considered having the base provide a default implementation of `groupHeaderValueChanged()` that called `applyPlaceholderTextIfNeeded()`, but then it needs to return a boolean for overrides to check to determine whether to do any further formatting. Maybe it's now justified to have this base class define `groupHeaderValueChanged()`, `columnConfigChanged()`, and `updateText()` (abstract or no-op):

```ts
    protected columnConfigChanged(): void {
        if (!this.applyPlaceholderTextIfNeeded()) {
            this.updateText();
        }
    }
    protected groupHeaderValueChanged(): void {
        if (!this.applyPlaceholderTextIfNeeded()) {
            this.updateText();
        }
    }
    protected abstract updateText(): void;
```